app/javascript/App/bundles/NewsfeedPage/NewsList.tsx

```diff
@@ -178,7 +149,7 @@ class NewsList extends React.Component<Props, State> {
             : {}
         }
       >
-        {this.renderView() /* viewState, initialRenderTips, readNewsIds, isLoading, fetchMoreNewsFeed */}
+        {this.renderView()}
```

Comment:
Just FYI: usually `renderSomething()` is a sign that you need new component, but not critical.